public/css/module.less

```diff
@@ -1737,7 +1749,7 @@ table.pivot {
   padding-left: 2.4em;
   line-height: 2em;
   text-decoration: none;
-  color: #777;
+  .var(color, gray-light);
```

Comment:
The contrast is too low in the tree view. Please use `-text-color` here.

![Screen Shot 2021-11-23 at 17 01 03](https://user-images.githubusercontent.com/6977434/143059253-6907b816-5ef9-43ea-9a18-584971eb8721.png)

![Screen Shot 2021-11-23 at 17 01 17](https://user-images.githubusercontent.com/6977434/143059302-454e565d-cc50-456c-9703-b4f85962fbb9.png)